src/components/Table/Table.tsx

```diff
@@ -3,47 +3,23 @@ import type { ReactNode } from 'react';
 import React from 'react';
 import styles from './Table.module.css';
 import TableBody from '../TableBody';
+import TableCaption from '../TableCaption';
 import TableCell from '../TableCell';
 import TableFooter from '../TableFooter';
 import TableHeader from '../TableHeader';
 import TableHeaderCell from '../TableHeaderCell';
 import TableRow from '../TableRow';
 
-export interface Props {
+export type Props = React.TableHTMLAttributes<HTMLTableElement> & {
   /**
-   * Behavior variations:
-   * - **overflow** renders a table with a max width to overflow
-   * - **stacked** renders a table that stacks its content into a card on small screens
+   * Child node(s) that can be nested inside component.
    */
-  behavior?: 'overflow' | 'stacked';
-  /**
-   * HTML caption property for table
-   */
-  caption: string;
```

Comment:
Removed in favor of `<TableCaption>` subcomponent, which is closer to native `table` element use with `caption` element